content/blog/2023-12-26-almalinux-container-cloud-live-media-updates.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "AlmaLinux Cloud, Container and Live Media Images Updates"
+type: blog
+author: 
+ name: "Andrew Lukoshko"
+ bio: "Release Engineering Lead"
+ image: /users/alukoshko.jpg
+date: 2023-12-26
+images:
+  - /blog-images/2023-12-21-images-updates.jpg
+post: 
+    title: "AlmaLinux Cloud, Container and Live Media Images Updates"
+    image: /blog-images/2023-12-21-images-updates.jpg
+---
+
+Hey Community! We're just sharing some news on both AlmaLinux 9.3 and 8.9 images updates. 
+
+
+## Cloud and Container Images Updates and Changes
+
+AlmaLinux OS provides a variety of cloud images, so we've been updating them to 9.3 and 8.9 versions including: 
+* [Amazon AWS](https://wiki.almalinux.org/cloud/AWS.html)
+    * Starting AlmaLinux OS versions 8.9 and 9.3 Amazon Machine images have UEFI boot support. Thus boot mode was changed to the `uefi-preferred` meaning that UEFI will be chosen on the supported instance types. We are planning to publish separate AMIs with enabled Secure Boot and NitroTPM(vTPM) support.
+* [Generic Cloud](https://wiki.almalinux.org/cloud/Generic-cloud.html)
+* [OpenNebula](https://wiki.almalinux.org/cloud/OpenNebula.html)
+    * Starting AlmaLinux OS 8.9 Generic Cloud and OpenNebula images have hybrid - BIOS and UEFI - boot support. The download URLs of the UEFI images are symlinked to the current image for compatibility.
+* [Microsoft Azure](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-x86_64)
```

Comment:
Can we add the AArch64 version of Azure offer too? 
https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-arm

Reply:
@LKHN please check